aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -620,6 +629,7 @@ module aptos_token::token {
         property_types: vector<String>
     ): TokenDataId acquires Collections {
         let account_addr = signer::address_of(account);
+        // TODO: Why not create the Collections resource if it doesn't exist for convenience?
```

Comment:
just to prevent accedental collection creation

Reply:
if you can just have

if (!exists<Collections>...) {
  move_to(..., Collections {});
}